setup.py

```diff
@@ -57,9 +57,7 @@ def read(fname):
         'pandas==2.0.2 ',
         'pytest==8.3.3',
         'scipy==1.14.1',
-        'psutil==6.1.0',
-        'fastparquet==2023.4.0' #Will drop support of fastparquet in future versions
```

Comment:
Awesome. Thanks so much 😄 . Can you also test if `pyarrow` package is needed too? I think it is not.